translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+        ],
+        "editedGuild": [
+          "A **{guild}** nevű guild konfigurációja sikeresen frissítve lett."
+        ],
+        "premiumGuild": [
+          "Ez az opció csak a prémium szervereken használható.\n",
+          "Ha többet szeretnél tudni, használd a '/premium információ' parancsot."
+        ]
+      }
+  },
+  "config guilds add-rank": {
+    "name": "rang-hozzáadása",
+    "description": "Regisztrál egy guild rangot a bottal",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild amelyiknél a rang lesz."
+      },
+      {
+        "name": "rang",
+        "description": "A regisztrálandó rang."
+      },
+      {
+        "name": "role",
+        "description": "A role ami a ranghoz lesz kötve. Hagyd üresen, ha nem akarsz role-t."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a  `/config guilds guild-hozzáadás` paranccsal."
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "rankNotFound": [
+        "A **{rank}** rang nem létezik a **{guild}** nevű guild-ban."
+      ],
+      "addedRank": [
+        "A **{rank}** rang a **{guild}** nevű guild-ban regisztrálva lett!"
+      ]
+    }
+  },
+  "config guilds edit-rank": {
+    "name": "rang-módosítás",
+    "description": "Megváltoztatja a role-t, ami a guild ranghoz van kötve.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild amelyiknél a rang van."
+      },
+      {
+        "name": "rank",
+        "description": "A módosítandó rang."
+      },
+      {
+        "name": "role",
+        "description": "A role, ami a ranghoz van kötve. Hagyd üresen, ha nem akarsz role-t."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadás` paranccsal."
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "rankNotFound": [
+        "A **{rank}** rang nem létezik a **{guild}** nevű guild-ban."
+      ],
+      "editedRank": [
+        "A **{rank}** rang a **{guild}** nevű guildban {state}-re lett módosítva!"
+      ],
+      "role": [
+        "átállítva {role.mention}"
+      ],
+      "noRole": [
+        "eltávolítva"
+      ]
+    }
+  },
+  "config guilds remove-rank": {
+    "name": "rang-törlése",
+    "description": "Töröl egy Hypixel guild rangját.",
+    "options": [
+      {
+        "name": "guild",
+        "description": "A guild amelyiknél a rang van."
+      },
+      {
+        "name": "rang",
+        "description": "A törölnivaló rang."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadása` paranccsal."
```

Comment:
The whole command name should be translated, including top level
Should also be in backticks for `this effect`